src/controllers/auth.js

```diff
@@ -0,0 +1,9 @@
+exports.callback = async (req, res) => {
+  res.json(req.user);
+};
+
+exports.signout = (req, res) => {
+  const { id } = req.user;
+  req.logout();
```

Comment:
You also need to call `req.session.destroy()` since you're using express-session.